package.json

```diff
@@ -46,7 +46,7 @@
     "mini-css-extract-plugin": "^0.8.0",
     "moment-locales-webpack-plugin": "^1.1.0",
     "ngtemplate-loader": "^2.0.1",
-    "node-sass": "4.12.0",
+    "node-sass": "4.14.1",
```

Comment:
`node-sass` always seems to need to be upgrade to work with newer versions of node...